mypy/test/testcheck.py

```diff
@@ -67,6 +68,11 @@
     'check-newtype.test',
 ]
 
+if hasattr(typed_ast, '__version__'):
+    version_info = tuple(map(int, typed_ast.__version__.split('.')))  # type: ignore
+    if version_info >= (0, 5, 7):
```

Comment:
https://github.com/dropbox/typed_ast/pull/16 contains an interface change, so it's actually going to be version 0.6.0 -- could you bump that here?